docs/development/debugging/x64dbg.md

```diff
@@ -0,0 +1,29 @@
+# x64dbg
+
+{% hint style="warning" %}
+if you are not experienced with x64dbg it its recommended to use [Visual Studio](visualstudio.md)
+{% endhint %}
+
+
+## Windows
+
+* Download [x64dbg](https://sourceforge.net/projects/x64dbg/files/latest/download)
+* Extract the `release` folder somewhere on your PC
+* Download the latest relase of [SycllaHide](https://github.com/x64dbg/ScyllaHide/releases/tag/v1.4)
+* Merge the contents of the `x64dbg` folder into the previously extracted `release` folder
+* Run `x96dbg.exe` and select `x64dbg` or run `x64/x64dbg.exe`
+
+## Linux
+
+{% hint style="warning" %}
+Debugging Northstar under Linux is not trivial due to the direct dependency on
+{% endhint %}
+
+Debugging Northstar under Linux is not trivial due to the direct dependency on Origin.
+
+To simplify the use of x64dbg and automate running Origin a community member has created a script.
+
+* clone `https://git.jandroegehoff.de/sentry/ns-gdb`
```

Comment:
If getting the script merged into tools org can be done within the next day we should do that and then link to it. Links to 3rd party git servers might be seen suspiciously by not-so-experienced devs ig.

If it takes longer than that, don't bother we'll update it after merge into tools org.

Reply:
script has since been reworked and merged into the tools org https://github.com/R2NorthstarTools/ns-linux-dbg